tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3697,15 +3639,13 @@ public function userTriesToEnableUserUsingTheProvisioningApi(
 	 * @return void
 	 * @throws Exception
 	 */
-	public function deleteTheUserUsingTheProvisioningApi(string $user):void {
-		$this->emptyLastHTTPStatusCodesArray();
-		$this->emptyLastOCSStatusCodesArray();
+	public function deleteTheUserUsingTheProvisioningApi(string $user):ResponseInterface {
```

Comment:
it now the chance to rename the function
```suggestion
	public function deleteUser(string $user):ResponseInterface {
```